apps/storybook/src/forms/Button.stories.tsx

```diff
@@ -13,92 +13,104 @@ export default {
 
 type Story = StoryObj<typeof Button>;
 
-const Variants: Story = {
+const sizes = ["sm", "md", "lg"] as const;
+const appearances = [
+  "default",
+  "primary",
+  "secondary",
+  "danger",
+  "danger-outline",
+] as const;
+
+export const Appearance: Story = {
+  args: {
+    children: "Button",
+  },
   render: (args) => (
-    <Flex flexDirection="column" gap="sm">
-      {variants.map(([variant, label]) => (
-        <Flex flexDirection="row" gap="sm" key={variant}>
+    <Flex>
+      {appearances.map((appearance) => (
+        <Flex alignItems="center" flexDirection="row" gap="sm" key={appearance}>
           {sizes.map((size) => (
-            <Button {...args} key={size} size={size} variant={variant}>
-              {label} - {size}
-            </Button>
+            <Button {...args} appearance={appearance} key={size} size={size} />
           ))}
-          <Button disabled {...args} variant={variant}>
-            Disabled
-          </Button>
         </Flex>
       ))}
     </Flex>
   ),
 };
 
-const sizes = ["sm", "md", "lg"] as const;
-const appearances = [
-  ["default", "Default"],
-  ["primary", "Primary"],
-  ["danger", "Danger"],
-  ["secondary", "Secondary"],
-] as const;
-const variants = [
-  ["solid", "Solid"],
-  ["outline", "Outline"],
-  ["subtle", "Subtle"],
-] as const;
-
-export const Basic: Story = {
-  ...Variants,
-};
-
-export const Primary: Story = {
-  ...Variants,
+export const Sizes: Story = {
   args: {
-    colorScheme: "primary",
+    children: "Button",
   },
+  render: (args) => (
+    <Flex flexDirection="row">
+      {sizes.map((size) => (
+        <Button {...args} key={size} size={size}>
+          {size}
+        </Button>
+      ))}
+    </Flex>
+  ),
 };
 
-export const Danger: Story = {
-  ...Variants,
+export const Disabled: Story = {
   args: {
-    colorScheme: "danger",
+    children: "Button",
+    disabled: true,
   },
+  render: (args) => (
+    <Flex flexDirection="row">
+      {sizes.map((size) => (
```

Comment:
can we showcase the disabled state of all the different appearances instead of the sizes here?
```suggestion
      {appearances.map((appearance) => (
```

Reply:
Got it bhai!